### PR TITLE
fix: auto brightness

### PR DESCRIPTION
**Pull request by mikejgray**
https://github.com/OpenVoiceOS/ovos-gui-plugin-shell-companion/blob/a58bf321695931621da152c8f58bafdcff163a3a/ovos_gui_plugin_shell_companion/__init__.py#L81

## Summary by CodeRabbit

- **New Features**
  - Improved brightness control logic with a consistent dimming setting.
  - Enhanced configuration management by integrating display settings directly into the BrightnessManager.
  - Default behavior for auto night mode is now set to enabled.

- **Bug Fixes**
  - Streamlined device-specific brightness settings for better usability.
  - Enhanced error handling and logging for brightness adjustments.

- **Chores**
  - Updated `.gitignore` to exclude `.DS_Store` files for cleaner version control.

### CODE REVIEW

**Comment by mikejgray** [comment]
Failing test seems unrelated to this change...

**Comment by NeonDaniel** [comment]
Can you rebase on `dev`? This looks like something that was fixed with https://github.com/OpenVoiceOS/ovos-gui-plugin-shell-companion/pull/7

File: ovos_gui_plugin_shell_companion/brightness.py [hunk 1]
```diff
@@ -260,12 +260,10 @@ def start_auto_night_mode(self, message=None):
...
             if self.sunset_time < date < self.sunrise_time:
                 LOG.debug("It is night time")
-                self.bus.emit(Message("phal.brightness.control.auto.night.mode.enabled"))
+                self.bus.emit(Message("speaker.extension.display.set.auto.nightmode", {"auto_nightmode": True}))
             else:
                 LOG.debug("It is day time")
-                # TODO - implement this message in shell / check if it exists
-                # i just made it up without checking
-                self.bus.emit(Message("phal.brightness.control.auto.night.mode.disabled"))
+                self.bus.emit(Message("speaker.extension.display.set.auto.nightmode", {"auto_nightmode": True}))
```
Comment (NeonDaniel) [inline ovos_gui_plugin_shell_companion/brightness.py]: Should this be `False` to disable nightmode?

**Review by NeonDaniel — COMMENTED** [review]
Left a comment; I'm also not sure how to effectively test this

File: ovos_gui_plugin_shell_companion/brightness.py (hunk 1 above)
Comment (mikejgray) [reply ovos_gui_plugin_shell_companion/brightness.py]: Good catch...I wrote this patch for another user who initially reported it was good, but I haven't tested myself. I installed this on my office Mark 2 and I'll report back in the morning. Right now night mode is enabled but that seems to never turn off when I turn it on...

Comment (JarbasAl) [reply ovos_gui_plugin_shell_companion/brightness.py]: that was the bug this is supposed to fix, it never comes out of night mode when it turns into daytime

Comment (mikejgray) [reply ovos_gui_plugin_shell_companion/brightness.py]: I'll let you know in the morning. ;)

**Comment by mikejgray** [comment]
@NeonDaniel @JarbasAl This got fixed in another PR, didn't it? Can I close this?

**Comment by goldyfruit** [comment]
Auto brightness doesn't seem to work for me.

**Comment by mikejgray** [comment]
@goldyfruit @NeonDaniel @JarbasAl I think these changes should resolve it. I was able to get auto dimming to happen just by creating the config file, with no code changes, but this will also squash some minor bugs along the way.

Expected behavior: `~/.config/OvosDisplay.conf` is created and populated. Before, if it didn't exist and neither did `/etc/xdg/OvosDisplay.conf`, the config would simply not load. This meant dimming never worked.

File: ovos_gui_plugin_shell_companion/brightness.py [hunk 2]
```diff
@@ -192,11 +192,6 @@ def auto_dim_timer(self):
         while self.auto_dim_enabled:
             time.sleep(60)
```
Comment (NeonDaniel) [inline ovos_gui_plugin_shell_companion/brightness.py]: While you're at it, `Event.wait` can achieve the same thing more efficiently

**Comment by JarbasAl** [comment]
please rebase on #31

**Comment by mikejgray** [comment]
#31 is better, use that

**Pull request closed without merging**